metaserver/src/main/scala/scala/meta/languageserver/compiler/ScalacProvider.scala

```diff
@@ -16,9 +15,7 @@ import org.langmeta.inputs.Input
 import org.langmeta.io.AbsolutePath
 
 /** Responsible for keeping fresh scalac global instances. */
-class ScalacProvider(
-    serverConfig: ServerConfig
```

Comment:
`serverConfig` was never used.

Reply:
Probably relic from the time we indexed the classpath in ScalacProvider